Assets/Script/Venue/BundleBackgroundManager.cs

```diff
@@ -81,31 +84,44 @@ public void ExportBackground()
                 assetBundleBuild.assetBundleName = fileName;
                 assetBundleBuild.assetNames = assetPaths;
 
-                BuildPipeline.BuildAssetBundles(Application.temporaryCachePath,
-                    new[]
+                List<BuildTarget> platforms = type == ExportType.AllPlatforms ? new() {BuildTarget.StandaloneLinux64, BuildTarget.StandaloneWindows, BuildTarget.StandaloneOSX} : new() {EditorUserBuildSettings.activeBuildTarget};
```

Comment:
Consider styling it like this
```suggestion
                var platforms = type == ExportType.AllPlatforms
                    ? new List<BuildTarget>
                    {
                        BuildTarget.StandaloneLinux64, BuildTarget.StandaloneWindows, BuildTarget.StandaloneOSX
                    }
                    : new List<BuildTarget>
                    {
                        EditorUserBuildSettings.activeBuildTarget
                    };
```